src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -142,7 +142,7 @@ export type GetOpportunityUserStakingDataOutput = {
 export type GetOpportunityIdsOutput = OpportunityId[]
 
 // TODO: This is not FDA-approved and should stop being consumed to make things a lot tidier without the added cholesterol
-export type EarnOpportunityType = {
+export type EarnOpportunityTypeBase = {
```

Comment:
We can now remove the `export`.

Reply:
9a92da2c90f797da0e7ba487bdbbf155c573e438